build throw error

I'm unable to build project:
node version - v7.7.0, v6.9.5

After `npm run build` get error: 
```
ERROR in ../~/css-loader!../~/sass-loader/lib/loader.js!../~/postcss-loader/lib!./styles/main.scss
Module not found: Error: Can't resolve 'file-loader' in '/Users/sasha/work/react-vouchers/app'
 @ ../~/css-loader!../~/sass-loader/lib/loader.js!../~/postcss-loader/lib!./styles/main.scss 6:1699-1720
 @ ./styles/main.scss
 @ ./components/index.jsx
 @ ./index.jsx

ERROR in ../~/css-loader!../~/sass-loader/lib/loader.js!../~/postcss-loader/lib!./styles/main.scss
Module not found: Error: Can't resolve 'file-loader' in '/Users/sasha/work/react-vouchers/app'
 @ ../~/css-loader!../~/sass-loader/lib/loader.js!../~/postcss-loader/lib!./styles/main.scss 6:1888-1909
 @ ./styles/main.scss
 @ ./components/index.jsx
 @ ./index.jsx

ERROR in ./styles/main.scss
Module build failed: ModuleNotFoundError: Module not found: Error: Can't resolve 'file-loader' in '/Users/sasha/work/react-vouchers/app'
    at factoryCallback (/Users/sasha/work/react-vouchers/node_modules/webpack/lib/Compilation.js:264:39)
    at factory (/Users/sasha/work/react-vouchers/node_modules/webpack/lib/NormalModuleFactory.js:247:20)
    at resolver (/Users/sasha/work/react-vouchers/node_modules/webpack/lib/NormalModuleFactory.js:65:21)
    at asyncLib.parallel (/Users/sasha/work/react-vouchers/node_modules/webpack/lib/NormalModuleFactory.js:202:22)
    at /Users/sasha/work/react-vouchers/node_modules/async/dist/async.js:3838:9
    at /Users/sasha/work/react-vouchers/node_modules/async/dist/async.js:421:16
    at iteratorCallback (/Users/sasha/work/react-vouchers/node_modules/async/dist/async.js:996:13)
    at /Users/sasha/work/react-vouchers/node_modules/async/dist/async.js:906:16
    at /Users/sasha/work/react-vouchers/node_modules/async/dist/async.js:3835:13
    at /Users/sasha/work/react-vouchers/node_modules/async/dist/async.js:1074:9
    at /Users/sasha/work/react-vouchers/node_modules/async/dist/async.js:421:16
    at iteratorCallback (/Users/sasha/work/react-vouchers/node_modules/async/dist/async.js:996:13)
    at /Users/sasha/work/react-vouchers/node_modules/async/dist/async.js:906:16
    at /Users/sasha/work/react-vouchers/node_modules/async/dist/async.js:1071:13
    at resolver.resolve (/Users/sasha/work/react-vouchers/node_modules/webpack/lib/NormalModuleFactory.js:273:20)
    at onResolved (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/Resolver.js:70:11)
    at loggingCallbackWrapper (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at afterInnerCallback (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/Resolver.js:138:10)
    at loggingCallbackWrapper (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at Resolver.applyPluginsAsyncSeriesBailResult1 (/Users/sasha/work/react-vouchers/node_modules/tapable/lib/Tapable.js:181:46)
    at innerCallback (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/Resolver.js:125:19)
    at loggingCallbackWrapper (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at /Users/sasha/work/react-vouchers/node_modules/tapable/lib/Tapable.js:283:15
    at /Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/UnsafeCachePlugin.js:38:4
    at loggingCallbackWrapper (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at afterInnerCallback (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/Resolver.js:138:10)
    at loggingCallbackWrapper (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)
    at Resolver.applyPluginsAsyncSeriesBailResult1 (/Users/sasha/work/react-vouchers/node_modules/tapable/lib/Tapable.js:181:46)
    at innerCallback (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/Resolver.js:125:19)
    at loggingCallbackWrapper (/Users/sasha/work/react-vouchers/node_modules/enhanced-resolve/lib/createInnerCallback.js:31:19)

ERROR in /Users/sasha/work/react-vouchers/node_modules/extract-text-webpack-plugin/loader.js??ref--2-0!/Users/sasha/work/react-vouchers/node_modules/style-loader/index.js!/Users/sasha/work/react-vouchers/node_modules/css-loader/index.js!/Users/sasha/work/react-vouchers/node_modules/sass-loader/lib/loader.js!/Users/sasha/work/react-vouchers/node_modules/postcss-loader/lib/index.js!/Users/sasha/work/react-vouchers/app/styles/main.scss doesn't export content
Child extract-text-webpack-plugin:
       [0] ../~/css-loader/lib/css-base.js 2.26 kB {0} [built]
       [1] ../~/css-loader!../~/sass-loader/lib/loader.js!../~/postcss-loader/lib!./styles/main.scss 2.14 kB {0} [built]
    
    ERROR in ../~/css-loader!../~/sass-loader/lib/loader.js!../~/postcss-loader/lib!./styles/main.scss
    Module not found: Error: Can't resolve 'file-loader' in '/Users/sasha/work/react-vouchers/app'
     @ ../~/css-loader!../~/sass-loader/lib/loader.js!../~/postcss-loader/lib!./styles/main.scss 6:1699-1720
    
    ERROR in ../~/css-loader!../~/sass-loader/lib/loader.js!../~/postcss-loader/lib!./styles/main.scss
    Module not found: Error: Can't resolve 'file-loader' in '/Users/sasha/work/react-vouchers/app'
     @ ../~/css-loader!../~/sass-loader/lib/loader.js!../~/postcss-loader/lib!./styles/main.scss 6:1888-1909
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,10 +5,16 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [0.0.3]
-- Added styles.
-- Added SVG buttons.
+
+### Added
+- Styles.
+- SVG buttons.
+
+### Fixed
+- Lost `file-loader` dependency in `package.json`.
 
 ## [0.0.2]
+
 ### Added
 - Vouchers fetch function.
 - Vouchers table.
@@ -16,6 +22,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 - Vouchers patch in `Reducer`.
 
 ## 0.0.1 - 2017-05-18
+
 ### Added
 - `WebPack` configuration files.
 - `Component`s with `Router`.

--- a/package.json
+++ b/package.json
@@ -52,6 +52,7 @@
   "dependencies": {
     "dateformat": "^2.0.0",
     "faker": "^4.1.0",
+    "file-loader": "^0.11.1",
     "history": "^3.0.0",
     "prop-types": "^15.5.10",
     "react": "^15.4.2",

--- a/package.json
+++ b/package.json
@@ -34,6 +34,7 @@
     "eslint-plugin-jsx-a11y": "^4.0.0",
     "eslint-plugin-react": "^6.10.3",
     "extract-text-webpack-plugin": "^2.1.0",
+    "file-loader": "^0.11.1",
     "ignore-styles": "^5.0.1",
     "jsdom": "^10.1.0",
     "jsdom-global": "^3.0.2",
@@ -52,7 +53,6 @@
   "dependencies": {
     "dateformat": "^2.0.0",
     "faker": "^4.1.0",
-    "file-loader": "^0.11.1",
     "history": "^3.0.0",
     "prop-types": "^15.5.10",
     "react": "^15.4.2",

